utils/R/do_conversions.R

```diff
@@ -53,7 +53,7 @@ do_conversions <- function(settings, overwrite.met = FALSE, overwrite.fia = FALS
     }
     
     # soil extraction
-    if(input.tag == "soil"){
+    if(input.tag == "soil"&& is.null(input$path)){
```

Comment:
I understand that, at the moment, there's a conflict between the soil tag in ED2 and the soil processing in do_conversions. However, in the long run, this is.null check needs to go -- the equivalent check in met.process caused many bugs when it was there and even more when it was finally removed. Developers need to not rely on this check to know whether the soil data being passed in is already in model-specific format, whether it's in it's raw format, or whether it's in the intermediate 'standard' format.